salt/state.py

```diff
@@ -2002,12 +2005,17 @@ def reconcile_procs(self, running):
                                'changes': {}}
                     try:
                         with salt.utils.fopen(ret_cache, 'rb') as fp_:
-                            ret = msgpack.loads(fp_.read())
+                            ret = msgpack.loads(fp_.read(), encoding='utf-8')
```

Comment:
I saw a few encoding-related constants but I wasn't sure which to use...
I added encoding=utf-8 so that the keys and values won't be binary strings so I can access stuff via ret['start_timestamp']